src/content/blog/why-support-ddev.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Why Should We Support DDEV Financially?"
+pubDate: 2023-05-24
+summary: "Explaining the value of DDEV that makes financial support of the project worth doing"
+author: Randy Fay
+featureImage:
+    src: /img/blog/2023/05/developers-at-table.png
+    alt: Image of a team of web developers working together at a table and delighting in the project they have in front of them
+categories:
+  - Community
+---
+
+The DDEV project is committed to setting itself to be sustainable for the long term. We know that your choice of a tool is a signficant investment of resources, and we want your choice to last and be rewarding for the long term. In order to ensure that, we're doing a fund drive, explaining to teams, organizations, and individuals why it's important to financially support the project. Our key priority right now is adding [another paid maintainer](recruiting-maintainers.md) and a documentation/marketing maintainer. We'll be be updating you regularly about this initiative.
+
+## What Sets DDEV Apart
+
+- **Installation**: DDEV is easy to install. It's a single binary backed by Docker providers everywhere.
+- **Project Setup**: Adding DDEV to a project is as easy as `ddev config` and can take as little as a minute.
+- **Reliability**: DDEV's behavior is predictable and reliable. Automated testing on all supported platforms and manual testing of every new feature make it rock solid.
```

Comment:
When i've first read that sentence 

> Automated testing on all supported platforms and manual testing of every new feature make it rock solid.

 i thought "make" refers to DDEV instead of the automated and manual tests. even though it is clear written correctly it is still somehow hard to read and process. but i haven't had an idea for an alternative suggestion yet.